src/error.rs

```diff
@@ -170,6 +188,21 @@ impl From<Error> for io::Error {
             }
         }
     }
+
+    #[cfg(not(feature = "std"))]
+    fn from(j: Error) -> Self {
```

Comment:
This function is never compiled. The surrounding impl block only exists if std is enabled. Please remove.